recipe/build.sh

```diff
@@ -10,3 +9,14 @@ bash configure  --prefix=$PREFIX $OPTS
 make
 make check
 make install
+
+if [ "$(uname)" == "Darwin" ]; then
+    mv nextstep/Emacs.app $PREFIX/Emacs.app
+    mkdir -p $PREFIX/bin
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs $PREFIX/bin/emacs-25.1
+    ln -s $PREFIX/bin/emacs-25.1 $PREFIX/bin/emacs
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/ctags $PREFIX/bin/ctags
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/ebrowse $PREFIX/bin/ebrowse
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/emacsclient $PREFIX/bin/emacsclient
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/etags $PREFIX/bin/etags
```

Comment:
Seems this was added in https://github.com/conda/conda-build/pull/869. It's been there for a while, though, so it seems weird that it would just start failing on symlinks now. Maybe the problem is that this is a recursive symlink.